SETUP.md

```diff
@@ -64,29 +64,35 @@ The Pretendo Network website uses this server as an API for querying user inform
+| `PN_ACT_CONFIG_CDN_SUBDOMAIN`                 | Subdomain used to serve CDN contents if s3 is disabled                                                          | Yes      |
+| `PN_ACT_CONFIG_CDN_DISK_PATH`                 | File system path used to store CDN contents if s3 is disabled                                                   | Yes      |
+| `PN_ACT_CONFIG_CDN_BASE_URL`                  | URL for serving CDN contents (usually the same as s3 endpoint)                                                  | No       |
+| `PN_ACT_CONFIG_WEBSITE_BASE`                  | Website URL                                                                                                     | Yes      |
+| `PN_ACT_CONFIG_AES_KEY`                       | AES-256 key used for encrypting tokens                                                                          | No       |
+| `PN_ACT_CONFIG_DATASTORE_SIGNATURE_SECRET`    | HMAC secret key (16 bytes in hex format) used to sign uploaded DataStore files                                  | No       |
+| `PN_ACT_CONFIG_GRPC_MASTER_API_KEY_ACCOUNT`   | Master API key to interact with the account gRPC service                                                        | No       |
+| `PN_ACT_CONFIG_GRPC_MASTER_API_KEY_API`       | Master API key to interact with the API gRPC service                                                            | No       |
+| `PN_ACT_CONFIG_GRPC_PORT`                     | gRPC server port                                                                                                | No       |
+| `PN_ACT_CONFIG_STRIPE_SECRET_KEY`             | Stripe API key. Used to cancel subscriptions when scrubbing PNIDs                                               | Yes      |
+| `PN_ACT_CONFIG_SERVER_ENVIRONMENT`            | Server environment. Currently only used by the Wii U Account Settings app. `prod`/`test`/`dev`                  | Yes      |
+| `PN_ACT_CONFIG_DISCOURSE_FORUM_URL`           | Used for anonymizing user accounts on Discourse during account deletion. Forum anonymization skipped if not set | No       |
+| `PN_ACT_CONFIG_DISCOURSE_API_KEY`             | Used for anonymizing user accounts on Discourse during account deletion. Forum anonymization skipped if not set | No       |
+| `PN_ACT_CONFIG_DISCOURSE_API_USERNAME`        | Used for anonymizing user accounts on Discourse during account deletion. Forum anonymization skipped if not set | No       |
+| `PN_ACT_CONFIG_GRPC_MIIVERSE_HOST`            | Used to remove Miiverse user data during account deletion                                                       | Yes      |
+| `PN_ACT_CONFIG_GRPC_MIIVERSE_PORT`            | Used to remove Miiverse user data during account deletion                                                       | Yes      |
+| `PN_ACT_CONFIG_GRPC_MIIVERSE_KEY_API`         | Used to remove Miiverse user data during account deletion                                                       | Yes      |
```

Comment:
Seems like the "optional" values are the other way around?
miiverse is not optional and its marked as optional, and forum is optional and marked as not optional.

Reply:
You're right, oops